tests/e2e/support/api/graph/userManagement.ts

```diff
@@ -80,6 +80,9 @@ export const createGroup = async ({
   })
 
   checkResponseStatus(response, 'Failed while creating group')
+
+  const responseData = await response.json()
+  group.uuid = responseData.id
```

Comment:
After creating via api -> I set groups uuid and can check that group is exist or not exist on the web